src/mui/button/SaveButton.js

```diff
@@ -31,21 +29,25 @@ export class SaveButton extends Component {
             submitOnEnter,
             redirect,
         } = this.props;
+        const successStyle = {
+          bg: '#3DCC91',
+          label: "#FFFFFF",
+        };
         const type = submitOnEnter ? 'submit' : 'button';
-        const ButtonComponent = raised ? RaisedButton : FlatButton;
         return (
-            <ButtonComponent
+            <RaisedButton
                 type={type}
                 label={label && translate(label, { _: label })}
                 icon={
                     saving && saving.redirect === redirect ? (
                         <CircularProgress size={25} thickness={2} />
                     ) : (
-                        <ContentSave />
+                        null
                     )
                 }
                 onClick={this.handleClick}
-                primary={!saving}
+                backgroundColor={successStyle.bg}
```

Comment:
The background will be the same color while saving ? 

It could be good to have a lighter blue

Reply:
it won't be blue anymore, the saveButton will be green